Interfaces/Website/index.html

```diff
@@ -388,7 +388,28 @@ <h1>: المواقع<i class="fa" aria-hidden="false"></i></h1>
                         </div>
                     </div>
 
-
+                    <div class="card text-center">
+                        <div class="card-header">
+                            القَارِئ
+                        </div>
+                        <div class="card-body">
+                            <p class="card-props">
+                                يقوم هذا النموذج بالتعرف على النص الموجود في الصور.
+                            </p>

```

Comment:
It would be useful if you could add more information like what model/libraries were used, etc. Check other model cards.